### PR TITLE
docs(class): enhance document for eg.Class

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#294

## Details
enhance document for eg.Class

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

**Comment by mixed** [comment]
👍

File: src/class.js [hunk 1]
```diff
@@ -54,11 +56,12 @@ eg.module("class", [eg], function(ns) {
 	};
 	/**
 	 * Extends class
-	 * @ko extend는 Class를 상속할 때 사용합니다.
+	 * @ko extend는 Class를 상속할 때 사용한다.
 	 * @static
 	 * @method eg.Class.extend
 	 * @param {eg.Class} oSuperClass Super class. <ko>상속하려는 클래스</ko>
```
Comment (netil) [inline src/class.js]: It should be take same way of expression.
There're comments referring in both ways. 'Class' and '클래스'

Comment (jongmoon) [inline src/class.js]: @netil I agree, Thank you!

File: src/class.js [hunk 2]
```diff
@@ -10,12 +10,14 @@ eg.module("class", [eg], function(ns) {
...
 	 * The Class object is used to implement object-oriented style programming
 	 * @group egjs
-	 * @ko Class는 어플리케이션을 객체지향 프로그래밍 방식으로 구현하는데 사용합니다.
+	 * @ko Class는 어플리케이션을 객체지향 프로그래밍 방식으로 구현하는데 사용한다.
 	 * @class
 	 * @name eg.Class
 	 *
 	 * @support {"ie": "7+", "ch" : "latest", "ff" : "latest",  "sf" : "latest", "edge" : "latest", "ios" : "7+", "an" : "2.1+ (except 3.x)"}
-	 * @param {Object} def Class definition of object literal type. <ko>리터럴 형태의 클래스 정의부</ko>
+	 * @param {Object} def Class definition of object literal type. <ko>Class 정의부로 Object 의 <a href="https://en.wikipedia.org/wiki/Literal_(computer_programming)">리터럴 규칙</a>을 따른다. 단, "construct" 는 생성자 함수를 위해 예약된 키 값이다. </ko>
```
Comment (sculove) [inline src/class.js]: 단, "construct" 는 생성자 함수를 위해 예약된 키 값이다 =>  단, "construct" 는 생성자 함수를 위한 예약 키 값이다

File: src/class.js [hunk 3]
```diff
@@ -54,12 +56,13 @@ eg.module("class", [eg], function(ns) {
...
 	 * Extends class
-	 * @ko extend는 Class를 상속할 때 사용합니다.
+	 * @ko extend는 Class를 상속할 때 사용한다.
 	 * @static
 	 * @method eg.Class.extend
-	 * @param {eg.Class} oSuperClass Super class. <ko>상속하려는 클래스</ko>
-	 * @param {Object} def Class definition of object literal type. <ko>리터럴 형태의 클래스 정의부</ko>
-	 * @return {eg.Class} instance of new eg.Class <ko>새로 생성된 eg.Class 인스턴스</ko>
+	 * @param {eg.Class} oSuperClass Super class. <ko>상속하려는 Class</ko>
+	 * @param {Object} def Class definition of object literal type. <ko>Class 정의부로 Object 의 <a href="https://en.wikipedia.org/wiki/Literal_(computer_programming)">리터럴 규칙</a>을 따른다. 단, "construct" 는 생성자 함수를 위해 예약된 키 값이다.</ko>
```
Comment (sculove) [inline src/class.js]: it's same case.
https://github.com/naver/egjs/pull/312/files#r71841844

**Comment by sculove** [comment]
LGTM

**Pull request merged**